### PR TITLE
Prepare styleguide for new landing page and new design

### PR DESCRIPTION
**Pull request by guusli**
Update according to the new design : Buttons, forms, colors, font

Add the footer to the styleguide

Extract the message with a button to the styleguide (ButtonMessage)

Add media query variables

### CODE REVIEW

File: src/mnd-bootstrap/components/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/components/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.PoptypeFooter__link {
+  display: block;
+  color: $text-color;
+  margin: $gutter-horizontal;
+
+
+  @media screen and (min-width: $screen-sm) {
+      display: inline;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Line should be indented 4 spaces, but was indented 6 spaces

File: src/mnd-bootstrap/components/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      margin: auto ($gutter-horizontal);
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Line should be indented 4 spaces, but was indented 6 spaces

File: src/mnd-bootstrap/components/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+*/
+
+.PoptypeFooter {
+  $footer-max-width: 1000px;
+
+  @include display(flex);
+  @include flex-direction(column);
+
+  max-width: $footer-max-width;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Properties should be ordered margin, max-width, padding

File: src/mnd-bootstrap/components/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    @include justify-content(flex-end);
+  }
+}
+
+.PoptypeFooter__links {
+  text-align: center;
+}
+
+.PoptypeFooter__link {
+  display: block;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Properties should be ordered color, display, margin

File: src/mnd-bootstrap/components/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  }
+
+  a,
+  span {
+    font-family: $font-family-sans-serif;
+    font-size: 16px;
+  }
+}
+
+.PoptypeFooter__copyright {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Selector `PoptypeFooter__copyright` should be written in lowercase with hyphens

File: src/mnd-bootstrap/components/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  text-align: center;
+
+  @media screen and (min-width: $screen-sm) {
+    margin: 0;
+    margin-right: $gutter-horizontal;
+    order: -1;
+  }
+}
+
+.PoptypeFooter__social {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Selector `PoptypeFooter__social` should be written in lowercase with hyphens

File: src/mnd-bootstrap/components/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+  text-align: center;
+
+  @media screen and (min-width: $screen-sm) {
+    @include display(flex);
+    @include flex(1);
+    @include justify-content(flex-end);
+  }
+}
+
+.PoptypeFooter__links {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Selector `PoptypeFooter__links` should be written in lowercase with hyphens

File: src/mnd-bootstrap/components/_footer.scss [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+  text-align: center;
+}
+
+.PoptypeFooter__link {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_footer.scss]: Selector `PoptypeFooter__link` should be written in lowercase with hyphens

**Comment by guusli** [comment]
screen shot 2016-10-25 at 15 58 09

screen shot 2016-10-25 at 16 09 29

screen shot 2016-10-25 at 16 10 11

File: doc_assets/styleguide.scss [hunk 10]
```diff
@@ -214,3 +214,27 @@ p.styleguide {
 .font-size-very-large {
   font-size: $font-size-very-large;
 }
+
```
Comment (guusli) [inline doc_assets/styleguide.scss]: I suggest we rename our colors to somthing more understandable 🙄

File: src/mnd-bootstrap/css/_forms.scss [hunk 11]
```diff
@@ -143,14 +144,14 @@ category: CSS
...
+  border: $form-border-width solid $gray-lighten;
 }
 
 .form-control {
   background: $white;
-  border: $border-width solid $gray-lighten;
+  border: $form-border-width solid $gray-lighten;
   box-shadow: none;
-  padding: $padding-base-vertical $padding-base-horizontal $input-bottom-padding;
+  padding: 7px  15px 8px;
```
Comment (guusli) [inline src/mnd-bootstrap/css/_forms.scss]: This should maybe be use varibles but I'm not sure if I like using global variables for a form component.

File: src/mnd-bootstrap/variables/_colors.scss [hunk 12]
```diff
@@ -70,20 +88,26 @@ Used for borders, background and dividers
...
+$gray-bluer: #647D8E;
 $gray-light: #999;
-$gray-lighten: #ddd;
-$gray-lighter: #eee;
-$gray-lightest: #fafafa;
-$gray-blue: #e0e3e9;
+$gray-lighten: #98adbb;
+$gray-lighter: #cbd7e4;
+$gray-lightest: #e0e3e9;
+$gray-lightestest: #F5F9FA;
```
Comment (guusli) [inline src/mnd-bootstrap/variables/_colors.scss]: 😫
I guess we can find a better naming scheme for our gray values

File: src/mnd-bootstrap/variables/_colors.scss [hunk 13]
```diff
@@ -70,20 +88,26 @@ Used for borders, background and dividers
 
 
 // Palette
-$color-1: #3eb991;
-$color-2: #ffd34f;
-$color-3: #f55065;
-$color-4: #0096d1;
+$color-1: #02ad55;
```
Comment (guusli) [inline src/mnd-bootstrap/variables/_colors.scss]: I suggest we rename our colors to somthing more understandable 🙄

Comment (guusli) [inline src/mnd-bootstrap/variables/_colors.scss]: I suggest we rename our colors to somthing more understandable 🙄

File: src/mnd-bootstrap/variables/_colors.scss (hunk 12 above)
Comment (dbackeus) [inline src/mnd-bootstrap/variables/_colors.scss]: 👍

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 14]
```diff
@@ -0,0 +1,81 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+    width: 768px;
+  }
+
+  &__left, &__right {
+    display: block;
+    margin-bottom: 10px;
+
+    @media (min-width: $screen-sm) {
+      display: table-cell;
+      vertical-align: middle;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: Properties should be ordered display, margin-bottom, vertical-align

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+    <a class="btn ButtonMessage__button" href="/">Try us right now</a>
+  </div>
+</div>
+```
+
+*/
+
+.ButtonMessage {
+  background: $green-icy;
+  border-radius: 5px;
```
Comment (bzf) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: Could use the `$border-radius` variable here

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+  display: table;
+  font-family: $font-family-sans-serif;
+  margin: $space-larger $space-large;
+  overflow: auto;
+  padding: 30px;
+  text-align: left;
+
+  @media screen and (min-width: 768px) {
```
Comment (bzf) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: Maybe wanna use the `$screen-md`?

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+    display: inline-block;
+    font-family: $font-family-sans-serif;
+    font-size: 16px;
+    line-height: 20px;
+    margin-bottom: 10px;
+    vertical-align: top;
+  }
+
+  &__button {
+
```
Comment (bzf) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: newline

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+    background: $color-1;
+    color: $white;
+    float: right;
+    font-weight: 500;
```
Comment (bzf) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: There is a `$semi-bold` variable but yeah.. 😄

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+    margin: 0;
+    padding: 7px 15px 8px;
+    width: 100%;
+
+    @media screen and (min-width: 768px) {
```
Comment (bzf) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: `$screen-md`?

File: src/mnd-bootstrap/components/_footer.scss [hunk 21]
```diff
@@ -0,0 +1,112 @@
...
+  display: block;
+  color: $text-color;
+  margin: $gutter-horizontal;
+
```
Comment (bzf) [inline src/mnd-bootstrap/components/_footer.scss]: newline

File: src/mnd-bootstrap/components/_buttonMessage.scss [hunk 22]
```diff
@@ -0,0 +1,80 @@
...
+  display: table;
+  font-family: $font-family-sans-serif;
+  margin: $space-larger $space-large;
+  overflow: auto;
+  padding: 30px;
+  text-align: left;
+
+  @media screen and (min-width: $screen-md) {
+    margin: 0 auto;
+    width: 100%
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_buttonMessage.scss]: Declaration should be terminated by a semicolon

**Pull request merged**